### DISCUSSION TITLE
Integration with google maps

### DISCUSSION THREAD

**Discussion by novakand** [visgl/deck.gl#5268] [Q&A]
I am engaged in the interaction with google maps, the question arises of how to make the IconLayer drag and drop
I also noticed that when dragging the map, the Icon Layer jerks a little. I will be glad to help. https://jsfiddle.net/Novakand/50uj7dbg/17/

**Comment by Pessimistress** [comment]
deck.gl does not control user interaction when used as a Google Maps overlay. You will have to register your own events and figure out how to block the map from panning. Once you obtain the cursor position you can call Deck’s picking methods to find out which icon is under the cursor and update the layer data.

**Comment by novakand** [comment]
Twitching of layers when dragging the map. Will this issue be resolved?
https://jsfiddle.net/Novakand/50uj7dbg/17/

**Reply by Pessimistress** [reply]
If you add `overflow: hidden` to `body` it will go away. I'm not convinced that this is a deck.gl issue as the canvas is positioned relative to the Maps panel container. Unfortunately, I don't have time to investigate further. It would be greatly appreciated if you want to look into this.
